acceptance/tests/wan-federation/wan_federation_test.go

```diff
@@ -8,17 +8,36 @@ import (
 	"fmt"
 	"strconv"
 	"testing"
+	"time"
 
+	terratestK8s "github.com/gruntwork-io/terratest/modules/k8s"
 	"github.com/hashicorp/consul-k8s/acceptance/framework/connhelper"
 	"github.com/hashicorp/consul-k8s/acceptance/framework/consul"
 	"github.com/hashicorp/consul-k8s/acceptance/framework/helpers"
 	"github.com/hashicorp/consul-k8s/acceptance/framework/k8s"
 	"github.com/hashicorp/consul-k8s/acceptance/framework/logger"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const StaticClientName = "static-client"
+const (
+	StaticClientName = "static-client"
```

Comment:
**nitpick:** Should probably be StaticClientName with a lowercase 's' (though you didn't do it).

Reply:
got rid of it, using the `connhelper` imported variable instead which is the same